workers.go

```diff
@@ -268,6 +270,33 @@ func (b WorkerSecretTextBinding) serialize(bindingName string) (workerBindingMet
 	}, nil, nil
 }
 
+type WorkerServiceBinding struct {
+	Service     string
+	Environment *string
+}
+
+func (b WorkerServiceBinding) Type() WorkerBindingType {
+	return WorkerServiceBindingType
+}
+
+func (b WorkerServiceBinding) serialize(bindingName string) (workerBindingMeta, workerBindingBodyWriter, error) {
+	if b.Service == "" {
+		return nil, nil, errors.Errorf(`Service for binding "%s" cannot be empty`, bindingName)
+	}
+
+	meta := workerBindingMeta{
+		"name":    bindingName,
+		"type":    b.Type(),
+		"service": b.Service,
+	}
+
+	if b.Environment != nil {
+		meta["environment"] = *b.Environment
```

Comment:
Oh, the pass as pointer required because this may be nil?

Reply:
Right, if you pass nil or empty string the binding is set to the default environment.